azurerm/data_source_data_lake_store_test.go

```diff
@@ -0,0 +1,96 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccDataSourceAzureRMDataLakeStore_payasyougo(t *testing.T) {
+	dataSourceName := "data.azurerm_data_lake_store.test"
+	rInt := acctest.RandIntRange(1, 999999)
```

Comment:
Could we use `acctest.RandInt()` like the majority of the other tests?

Reply:
there is a limitation on the name for an ADLS account which has to be between 3  to 24 characters. I wanted to restrict the length of the name to within that limit. RandInt() generates a larger string if I am not mistaken.